lib/CL/pocl_llvm_build.cc

```diff
@@ -418,14 +418,19 @@ int pocl_llvm_build_program(cl_program program,
 
   std::string AllBuildOpts = ss.str();
 
+  // Convert the -g to more specific debug options understood by the frontend.
+  AllBuildOpts = std::regex_replace(
+      AllBuildOpts, std::regex("-g(\\s+)"),
```

Comment:
AFAICT this regex needs at least a whitespace before, otherwise it could match things we don't want to match, e.g.
 `... -I/some/path/ending/with/letter-g ...`. And ideally we should not use regex for this kind of replacement at all, so a TODO would be nice here...